build.gradle

```diff
@@ -14,21 +14,22 @@
  * limitations under the License.
  */
 
-import com.github.spotbugs.SpotBugsTask
+
+import com.github.spotbugs.snom.SpotBugsReport
 import com.netflix.spinnaker.gradle.codestyle.SpinnakerCodeStylePlugin
 
 buildscript {
   ext {
     versions = [
-      lombok: '1.18.10',
-      resilience4j: '1.1.0',
-      awsSdkVersion: '1.11.847',
-      kork: '6.22.1',
-      guava: '28.1-jre',
-      springBoot: springBootVersion,
+      lombok: '1.18.16',
+      resilience4j: '1.6.1',
+      awsSdkVersion: '1.11.914',
+      kork: '7.90.0',
+      guava: '30.0-jre',
+      springBoot: '2.4.0',
```

Comment:
We should discuss Spring. We need to include it in two locations (project root build.gradle and the cerberus-web build.gradle). Before we were using a constant but Dependabot does not know how to parse constant versions. In order to enable auto upgrades for Spring the version has to be in line. 

This is less convenient from a developer perspective (we need to update Spring in 2 places instead of one but if we rely on Dependabot it should handle it for us).

Reply:
Which approach are we going with for this PR?